#!/usr/bin/env python
PACKAGE = "mantis_controller_joints"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen = ParameterGenerator()

gen.add("joint1", double_t, 0, "Joint 1 goal in radians", 0.0, -math.pi/2, math.pi/2)
gen.add("joint2", double_t, 0, "Joint 2 goal in radians", 0.0, -math.pi/2, math.pi/2)

exit(gen.generate(PACKAGE, "mantis_controller_joints_config_node", "JointGoals"))
